sdk/bare/common/drv/bsp.c

```diff
@@ -1,18 +1,34 @@
-#include "drv/bsp.h"
+#include "usr/user_defines.h"
+
+#if (HARDWARE_TARGET == 3) || (HARDWARE_TARGET == 4)
```

Comment:
Instead of using 3 (which implies REV C) and 4 (which implies REV D), we could define:
> #define AMDC_REV_C 3
> #define AMDC_REV_D 4

and then compare `HARDWARE_TARGET` to `AMDC_REV_C` or `AMDC_REV_D`, i.e.

> #if (HARDWARE_TARGET == AMDC_REV_C) || (HARDWARE_TARGET == AMDC_REV_D)

Thoughts on this?

Reply:
Yes, I waffled on this for a while. I do want to have defines like you say, but I'm unsure where to put them.... i.e. where in the code do we define `AMDC_REV_D` ... `drv/targets.h` ? perhaps that makes sense.

Reply:
Yeah, I like the idea of creating a `drv/targets.h` file. Ultimately, we may have more advanced `#ifdef`s in there, as hardware designs mature.

Reply:
Fixed in e87f98940fc4fe7bd4e6d045465da23af506960d